src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -126,11 +124,13 @@ namespace
 
         void on_tile_begin(
             const Frame&                frame,
-            Tile&                       tile,
+            foundation::Tile&           tile,
```

Comment:
You're in the .cpp file and there's a `using namespace foundation` statement so you can remove the qualification here (and everywhere in the file).